v2.0/enable-node-map.md

```diff
@@ -4,4 +4,199 @@ summary: Learn how to enable the node map in the Admin UI.
 toc: false
 ---
 
-*Docs coming soon*
\ No newline at end of file
+The **Node Map** visualizes the geographical configuration of a multi-regional cluster by plotting the node localities on a world map. The **Node Map** also provides real-time cluster metrics, with the ability to drill down to individual nodes to monitor and troubleshoot the cluster health and performance.
+
+This page walks you through the process of setting up and enabling the **Node Map**.
+
+{{site.data.alerts.callout_info}}The <b>Node Map</b> is an <a href="enterprise-licensing.html">enterprise-only</a> feature. However, you can <a href="https://www.cockroachlabs.com/pricing/request-a-license/">request a trial license</a>  to enable the <b>Node Map</b>. {{site.data.alerts.end}}
+
+<div id="toc"></div>
+
+## Setting Up and Enabling the Node Map
+
+To enable the **Node Map**, you need to start the cluster with the correct `--locality` flags and assign the latitudes and longitudes for each locality.
+
+{{site.data.alerts.callout_info}}The <b>Node Map</b> won't be displayed until <i>all</i> nodes are started with the correct <code>--locality</code> flags and all localities are assigned the corresponding latitudes and longitudes. {{site.data.alerts.end}}
+
+Consider a scenario of a four-node geo-distributed cluster with the following configuration:
+
+|  Node | Region | Datacenter |
+|  ------ | ------ | ------ |
+|  Node1 | us-east-1 | us-east-1a |
+|  Node2 | us-east-1 | us-east-1b |
+|  Node3 | us-west-1 | us-west-1a |
+|  Node4 | eu-west-1 | eu-west-1a |
+
+### Step 1. Ensure the CockroachDB Version is 2.0 or Higher
+
+~~~ shell
+$ cockroach version
+~~~
+
+If not, [upgrade to CockroachDB v2.0](upgrade-cockroach-version.html).
+
+### Step 2. Start (or Restart) the Nodes with the Correct `--locality` Flags
+
+If you have a cluster running without the `--locality` flags set, restart the nodes with the correct `--locality` flags(add link to rolling restart page once it's created).
```

Comment:
I'm not sure I'll get to create a rolling restart page in time for the release, so let's remove that parenthetical for now.

Reply:
I'm also starting to think that we might just want to direct users to our manual deployment page for a cluster startup example featuring `--locality` (it's not there yet, but I have a PR to update all of our deployment tutorials with `--locality`): http://www.cockroachlabs.com/docs/dev/deploy-cockroachdb-on-premises.html#step-3-start-nodes

If you'd rather leave these examples, then you should look at the manual deployment node startup commands and emulate them (plus `--locality`). In that case, in your examples, you shouldn't have `--store`, `--port`, or `--http-port`. Those aren't necessary in a real, distributed cluster. And you do need to add `----cache=25%` and `--max-sql-memory=25%`.

Reply:
Not even `--port`?